backend/actions.py

```diff
@@ -147,18 +146,33 @@ def hankering(self):
 
   def apply_effect(self):
     for target in self.targets:
-      target.loyalty += copysign(self.heal, self.subject.loyalty)
+      target.heal(self.heal * self.subject.level)
 
   def get_info(self):
     return {**super().get_info(),
             'heal': self.heal}
 
 
+class EdibleWildlife(Action):
+  def apply_effect(self):
+    self.subject.heal(0.01 * self.subject.influence)
+
+class SafetyCollar(Action):
+  inspiration = 3
+  def hankering(self):
+    return 4 if self.target else 0
+  def prepare(self, state):
+    self.target = self.subject.find_closest_opponent(state)
+  def apply_effect(self):
+    if target:
+      target.status.append({'type': 'SafetyCollar', 'damage': self.subject.influence * 5})
```

Comment:
I'm probably just missing this, but where is status initialized?
I don't mind too much, but why not just a safety_collar instance variable with the total amount of collar damage? I mean why this general status machinery, do you plan stuff like clear all status?

Reply:
> I'm probably just missing this, but where is status initialized?

In the constructor. I didn't add it. It was already there.

> I don't mind too much, but why not just a safety_collar instance variable with the total amount of collar damage? I mean why this general status machinery, do you plan stuff like clear all status?

I think the status stuff is also sent to the frontend, so we can graphically indicate this stuff. I like it. I'm doing the mushrooms with this too.

I didn't think adding random instance variables to Hero was fair play. But it's great if it is! 😄